heat/core/types.py

```diff
@@ -286,7 +286,7 @@ def char(cls):
     "c8": complex64,
     "c16": complex128,
     # numpy types
-    np.bool: bool,
+    np.bool_: bool,
```

Comment:
what is the difference with this vs `np.bool`?

Reply:
np.bool is deprecated. builtin `bool` or `np.bool_`should be used now.

Reply:
okay cool. this im fine with all of this